Makefile

```diff
@@ -359,10 +359,6 @@ BMH_NAMESPACE           ?= ${NAMESPACE}
 BAREMETAL_OS_CONTAINER_IMG ?=
 
 # Dataplane Operator
-DATAPLANE_IMG                                    ?= quay.io/openstack-k8s-operators/dataplane-operator-index:${OPENSTACK_K8S_TAG}
-DATAPLANE_REPO                                   ?= https://github.com/openstack-k8s-operators/dataplane-operator.git
-DATAPLANE_BRANCH                                 ?= ${OPENSTACK_K8S_BRANCH}
-DATAPLANE_COMMIT_HASH                            ?=
```

Comment:
There are couple of hits in ci-framework to these env vars. I'm not sure if that is active framework code or just some ci-framework self testing code https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-framework%20DATAPLANE_IMG&type=code